### PR TITLE
Adding some additional functionality

### PR DESCRIPTION
**Pull request by hurtstotouchfire**
Addresses:
- [x] try to get the html comments with github usernames included
- [x] support title as well as affiliation

### CODE REVIEW

File: _includes/earlier_signatures.html [hunk 1]
```diff
@@ -6,9 +6,6 @@
 
 <!-- Please don't add new signatures to this file. -->
 
-<li><a href="https://github.com/acouch">Aaron Couch, DKAN</a> <!-- from github user acouch -->
-<li>Aaron Kraus, Reciprocity, Inc. <!-- from verified email -->
```
Comment (emhoracek) [inline _includes/earlier_signatures.html:10]: Watch out, you lost some signatures here!

File: _layouts/index.html [hunk 2]
```diff
@@ -17,7 +17,14 @@
...
       , {{ signature.affiliation }}
       {% endif %}
+
+      {% if signature.title %}
+      , {{ signature.title }}
+      {% endif %}
     </li>
+    {% if signature.github %}
+    <!-- Github user: {{ signature.github }} -->
+    {% endif %}
```
Comment (emhoracek) [inline _layouts/index.html:27]: Can we do title, then affiliation? People see to be preferring that in their signatures.

File: _includes/earlier_signatures.html (hunk 1 above)
Comment (emhoracek) [reply _includes/earlier_signatures.html:10]: Oh cool, you're using them as tests. 😎

File: template.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+  name: Your name
+  link: A link for your name. (Remove this line if you don't want a link.)
+  affiliation: Your affiliation. (Remove this line if you don't want an affiliation.)
+  title: My title. (Remove this line if you don't want to list your title.)
+  github: Your github username (This won't show on the page, but it helps us keep track of where signatures originated in case we have to commit your signature for you to avoid merge conflicts.)
```
Comment (emhoracek) [inline template.md:6]: I can't imagine a case where there would be merge conflicts here. Two people making a file with the same name? But fine to leave in.

**Review by emhoracek — CHANGES_REQUESTED** [review]

File: _layouts/index.html (hunk 2 above)
Comment (emhoracek) [reply _layouts/index.html:27]: Meh, we can always change it later

**Review by emhoracek — APPROVED** [review]

File: _layouts/index.html (hunk 2 above)
Comment (hurtstotouchfire) [reply _layouts/index.html:27]: shit, yes that's what I meant to do. Can you commit on top, @emhoracek?

Comment (emhoracek) [reply _layouts/index.html:27]: no, I've really got to get back to work, but maybe make an issue later? It'll be an easy fix once the PR is already merged.

**Pull request merged**